test/test_python.py

```diff
@@ -326,18 +331,6 @@ def pytest_generate_tests(metafunc):
                              ids=test_modules_ids)
 
 
-def test_recursive_repr(selenium):
```

Comment:
Python 3.7 no longer raises a `RecursionError` when printing the `repr` of a recursive dictionary.  Pretty cool!